package.json

```diff
@@ -11,7 +11,7 @@
 		"yo-doctor": "lib/cli.js"
 	},
 	"engines": {
-		"node": ">=8"
+		"node": ">=12"
```

Comment:
```suggestion
		"node": ">=12.10.0"
```

Make align this field with other packages. They are using `>=12.10.0` for `node` runtime.

### FYI

- [`yo/package.json`](https://github.com/yeoman/yo/blob/main/package.json)
- [`generator/package.json`](https://github.com/yeoman/generator/blob/main/package.json)